### PR TITLE
[DEVX-1868] Add cypress-grep example

### PR DESCRIPTION
**Pull request by tianfeng92**
Add cypress-grep example

### CODE REVIEW

File: v1/examples/cypress-grep/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Filtering tests by tags
+```bash
+saucectl run --env "CYPRESS_grepTags=@mytags"
+```
+
+Filtering tests by test title
+```bash
+saucectl run --env "CYPRESS_grep=hello"
+```
```
Comment (alexplischke) [inline v1/examples/cypress-grep/README.md:50]: Does this work? Considering that the env var key is lowercased?

File: v1/examples/cypress-grep/cypress/e2e/examples/actions.cy.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+context('Actions', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/actions')
+  })
+  it('.type() - type into a DOM element', { tags: ['smoke'] }, () => {
+    // https://on.cypress.io/type
+    cy.get('.action-email')
+        .type('fake@email.com').should('have.value', Cypress.env('grepTags'))
```
Comment (alexplischke) [inline v1/examples/cypress-grep/cypress/e2e/examples/actions.cy.js:8]: That looks like a mistake?

```suggestion
        .type('fake@email.com').should('have.value', 'fake@email.com')
```

File: v1/examples/cypress-grep/README.md (hunk 1 above)
Comment (tianfeng92) [reply v1/examples/cypress-grep/README.md:50]: yep. This works actually.
Mac: https://app.saucelabs.com/tests/d606a8c9ef004df69309a6434386ddbd
Windows: https://app.saucelabs.com/tests/1dc80ef7431e4d42beb6a59f38d64b00

File: .github/workflows/test-v1alpha.yml [hunk 3]
```diff
@@ -103,3 +103,28 @@ jobs:
         run: |
           npm ci
           saucectl run
+  allure-plugin:
```
Comment (alexplischke) [inline .github/workflows/test-v1alpha.yml:106]: Why is this called allure-plugin? 🤔

Comment (tianfeng92) [reply .github/workflows/test-v1alpha.yml]: 😭 typo

**Review by alexplischke — APPROVED** [review]

**Pull request merged**